### ISSUE TITLE
Add a space between styles and subclasses

### ISSUE THREAD

**Issue by danilvalov** [csscomb/csscomb.js#441]
I have a code:

```
.class {
  width: 100px;
  height: 100px;
  .subclass {
    color: red;
  }
}
```

I want add a space between `height: 100px;` and `.subclass {`.

Example:

```
.class {
  width: 100px;
  height: 100px;
                                      <- indent
  .subclass {
    color: red;
  }
}
```

Can I do it?
##

**Comment by VictorCamargo** [comment]
+1

**Comment by dsongman** [comment]
+1

**Comment by vincentaudebert** [comment]
I think this is a duplicate of https://github.com/csscomb/csscomb.js/issues/210
And there is a PR for this: https://github.com/csscomb/csscomb.js/pull/405

It is currently on dev branch.

**Comment by VictorCamargo** [comment]
@vincentaudebert Didn't see this before, really thank you.

**Comment by VictorCamargo** [comment]
@vincentaudebert, sorry for that but, I did not set this up. Can you help me use it?

**Comment by jdalton** [comment]
Closed by #436.
